fcl/dunefd/gen/supernova/prodmarley_nue_cc_flat_radiological_decay0_dune10kt_halfActiveVol_1x2x6.fcl

```diff
@@ -3,4 +3,6 @@
 
 physics.producers.marley:      @local::dune_marley_nue_cc_flat_halfActiveVol
 
+physics.simulate: [ rns, marley, @sequence::dunefd_hd_backgrounds_1x2x6_v3_6 ]
```

Comment:
did this fcl not simulate backgrounds before?

Reply:
This fhicl is new, actually... But I did have to add this line after the first commit to change from the low activity that it inherited.